python-main.js

```diff
@@ -357,6 +357,10 @@ function web_editor(config) {
             return alert(config.translate.alerts.empty);
         }
         if (isModule) {
+            if (micropythonFs.getStorageRemaining() < 0){
+                micropythonFs.remove(filename);
+                return alert(config.translate.alerts.out_of_space);
```

Comment:
Should this be `module_out_of_space`?
Either way, one of the two new entries in editor.html could be removed.